### DISCUSSION TITLE
Login As

### DISCUSSION THREAD

**Discussion by djbbiz** [cortexrd/Knack-Toolkit-Library#25] [Ideas]
An ability to log in as another user with the click of a button would help the Admin troubleshoot issues on the user's side.

**Comment by cortexrd** [comment]
Hi Dean,

I don't think that this is possible in actual the Knack framework.  The app owner must create an account for the troubleshooter and he must log-in with this.

**But** once, in the App, it could be possible for that person to access a special page where he can create a few temporary accounts for debugging purposes, with various roles of interest, and that he can delete once done.

Norm

**Comment by CSWinnall** [comment]
Hi Dean 

We do this using a Dummy user which we can assign different user roles depending on what we are testing.

Craig

**Reply by djbbiz** [reply]
Yes I agree, in 10 years of working with Knack this has been my goto. But what you can't see is all the records that are specifically assigned to them through various permissions and connections. Roles are one way to connect data to accounts but there are other ways and especially if you have thousands of lines of custom javascript you may need to actually log in as them to verify what they are saying is happening. "Login As" has become standard in many SaaS apps now.

**Reply by MickGraham23** [reply]
yes, Quickbase has had it for over 10 years!
